packages/verification-vue-component/src/locales/en.json

```diff
@@ -65,12 +65,16 @@
                 "signedVia": "Signed via",
                 "issuerAddress": "Issuer Public Key",
                 "documentHash": "Document Hash",
+                "txHash": "Transaction Hash",
                 "registrationDate": "Issue Date & Time",
                 "registrationTransaction": "Registration Transaction",
                 "revocationTransaction": "Revocation Transaction",
                 "revocationDate": "Revocation Date & Time",
                 "smartContractAddress": "Smart Contract Address",
-                "signatureTransactions": "Signature transactions",
+                "signatureDetails": "Signature details",
+                "signatureLevel": "Signature Level",
+                "jurisdiction": "Jurisdiction",
+                "downloadSignature": "Download Signature",
```

Comment:
@stephalba is the english translation ok?

![grafik](https://user-images.githubusercontent.com/3669176/125458039-e041d0e3-4f98-487c-b6e5-d500ef2d1831.png)

Reply:
I would bot capitalize the second word (even if some of them are in all caps).

Reply:
Ok makes sense, should I change that also for all the other labels in the `meta` section?

Reply:
Yes please.

Reply:
Done